plugins/reporters/web-app/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ plugins {
     id("ort-library-conventions")
 }
 
-val generatedResourcesDir = file("$buildDir/generated-resources/main")
+val generatedResourcesDir = layout.buildDirectory.dir("generated-resources/main")
```

Comment:
Just wondering `.get().asFile` isn't needed?

Reply:
It's not needed here as `into()` below also accepts `Provider<...>`.